src/main/java/com/sorting/Selection.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * File: Selection.java
+ * Date: 10 Dec, 2015
+ * Author: Tarun Jain
+ * -----------------------------------------------
+ * Sorting is the process of rearranging a sequence of objects so as to put them 
+ * in some logical order. The selection sort algorithm sorts an array by repeatedly 
+ * finding the minimum element (considering ascending order) from unsorted part and 
+ * putting it at the beginning. It exchange one for every pass through the list. 
+ * 
+ *  *  Features:
+ *  
+ *  1. Selection Sort is implemented as an array of Comparable Objects which sorts
+ *     the objects in ascending order.
+ *  2. It is a generic implementation which can take any type of array of objects.   
+ *  3. The implementation is based on array based data structure.
+ *  4. The running time complexity of each operation is constant, O(n2).
+ *  
+ */
+package com.sorting;
+
+import java.util.Comparator;
+
+public class Selection {
+	private Selection() {
```

Comment:
Give /* */ comment on private methods. These comments are not read by javadoc. And write constructor is made private to prevent instantiation of this class of static methods.